view.rb

```diff
@@ -1,6 +1,30 @@
 class View
-  def initialize()
 
+  def welcome
+    print "Welcome to Flashcards! Once you see the definition, enter your guess or quit to end the game.\n"
+    print '*' * 83
   end
 
-end
\ No newline at end of file
+  def question_prompt
+    print 'Definition:  '
+  end
+
+  def answer_prompt
+    print "Answer: "
+  end
+
+  def input
+    gets#.chomp?
```

Comment:
@lksevans12 And sorry, I guess I missed your first comment -- .chomp would definitely be good here, so that our inputs don't have a newline appended at the end.